src/hct_mis_api/apps/dashboard/services.py

```diff
@@ -229,9 +229,7 @@ def refresh_data(cls, business_area_slug: str = "global") -> ReturnDict:
             Payment.objects.using("read_only")
             .select_related(
                 "business_area",
-                "program",
```

Comment:
why are we removing this?

Reply:
if is referenced in queries

Reply:
Program and fsp desegregation is not needed on the global dashboard